client/web/src/components/Datum/Datum.js

```diff
@@ -0,0 +1,58 @@
+import React from 'react';
+
+import Aux from '../../hoc/Aux/Aux';
+
+import styles from './Datum.module.css';
+
+const datum = (props) => {
+    let datum = null;
```

Comment:
The ` = null` is unnecessary

Reply:
No it's not. Javascript objects are initialized as undefined. This can through errors in the code. Null is better.